cypress-tests/cypress/support/commands.js

```diff
@@ -963,26 +1110,37 @@ Cypress.Commands.add(
       logRequestId(response.headers["x-request-id"]);
       expect(response.headers["content-type"]).to.include("application/json");
       if (response.status === 200) {
+        const recurringEnabledFromConfirm =
+          globalState.get("recurring_enabled");
+        if (
+          req_data.customer_acceptance === "object" &&
```

Comment:
Yes we have to check recurringEnabled is true or false for mandate payments. And these checks are based on multiple factors(In confirm customer acceptance passed/not and in create its a off_session payment or not then it will be true). So to handle that i made these changes.

Reply:
Done